lessons/0_workflow/workflow.html

```diff
@@ -0,0 +1,120 @@
+<html>
+
+<head>
+  <title>Javascript orientation session!</title>
+  <link rel="stylesheet" href="../../reveal.js/css/reveal.css" />
+  <link rel="stylesheet" href="../../reveal.js/css/theme/moon.css" />
+  <link rel="stylesheet" href="../../reveal.js/lib/css/zenburn.css" />
+  <link rel="JavaScript" href="../../reveal.js/plugin/markdown/marked.js" />
+  <link rel="JavaScript" href="../../reveal.js/plugin/markdown/markdown.js" />
+  <style>
+    .reveal pre {
+      margin: 0;
+    }
+
+    .monospace {
+      font-family: monospace !important;
+    }
+
+    .illustration {
+      border: 0px;
+    }
+
+    .reveal li {
+      margin-bottom: 0.5em;
+    }
+
+    .reveal pre {
+      width: 100%;
+    }
+
+    .reveal pre code {
+      cursor: pointer;
+    }
+
+    .reveal li pre {
+      width: fit-content;
+    }
+
+    .logo {
+      position: fixed;
+      /* Sit on top of the page content */
+      z-index: 2;
+      /* Specify a stack order in case you're using a different order for other elements */
+    }
+
+    .great {
+      color: green
+    }
+  </style>
+</head>
+
+<body>
+  <a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo" /></a>
+  <div class="reveal">
+    <div class="slides">
+      <section>
+        <h1>Workflow</h1>
+      </section>
+      <section>
+        <h3>Steps to upload your homework</h3>
+        <ul class="fragment">
+          <li>Create a NEW branch for your homework.</li>
+          <p>
+            <pre><code>git branch homework-two</code></pre>
```

Comment:
I taught my students how to create branches when checkout out a new branch. 
`$ git checkout -b homework_1` seems simpler

Reply:
and new branch should be done from master and not from any other random branch. Maybe we need to add 
`git branch` command there as well? So they first check that they are on master branch?
And actually `git fetch` `git pull origin master` would be helpful here, to get a fresh master from origin.

Reply:
yes, usually I use this command `git checkout -b homework_1` but I wanted to be consistent with the slides.

Reply:
sure, I can be more specific with the commands, it would be helpful for them.